AirCasting/SessionViews/CharEntriesCreator.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 05/05/2021.
+//
+
+import Foundation
+import Charts
+
+class ChartEntriesCreator {
+    var entries: [ChartDataEntry] = []
+    var stream: MeasurementStreamEntity
+    lazy var timeUnit: Double = stream.session.type == .mobile ? 60 : 60*60
+    lazy var lastMeasurementTime = {
+        self.stream.lastMeasurementTime
+    }
+    
+    init(stream: MeasurementStreamEntity) {
+        self.stream = stream
+    }
+    
+    func generateEntries() -> [ChartDataEntry] {
```

Comment:
seams like there is no usages of any properties from the class. so maybe changing the api From `generateEntries` to accept streams stream. `func generateEntries(for stream: MeasurementStreamEntity) -> [ChartDataEntry]` could make the class more flexible? in theory the time units could be adjusted and also a single instance of the class could exist.